tech/languages/c/cpp_installation.md

```diff
@@ -37,6 +37,13 @@ To view all the options that G++ can use, visit the manual page by typing:
 
 You will see that the manual page is identical with the one shown for GCC.
 
+
+## To install complete C/C++ development environment:
+
+```
+# sudo dnf groupinstall "C Development Tools and Libraries" "Development Tools"
```

Comment:
I'm not sure `C/Dev`+`Dev/tools` group and `GCC`  are that much related. It's more like bunch of libraries and debug tools for every compiler/lang. Would you mind to elaborate?

Reply:
As I mentioned it as **Complete developement environment** it will install bunch of libraries and debug tools and the user don't have to see xyz tool is missing. Its just an option if anyone need them all in one setup.

Reply:
Yes, it does, but it's not needed for developing C/C++ apps. This should go into standalone page, because otherwise it would go into almost all compiled languages pages.

Reply:
okay then

Reply:
Can you remove it then?